website/docs/d/service.html.markdown

```diff
@@ -39,5 +39,6 @@ The following arguments are supported:
 
 * `id` - The ID of the found service.
 * `name` - The short name of the found service.
+* `type` - The type for passing to a service dependency.
```

Comment:
Same as above. Technically, the types are for a variety of things and not just setting service dependency.